### PR TITLE
PA-7617-PA-7622 added new properties to Vehicle model

### PR DESCRIPTION
**Pull request by SvitlanaLitovchenko**
#### :tickets: Jira Tickets
https://1mojio.atlassian.net/browse/PA-7668
#### :interrobang: What does this PR do?
Added new properties to Vehicle model 
#### :world_map: Where should the reviewer start?
Vehicle class

### CODE REVIEW

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ObjectType.java [hunk 1]
```diff
@@ -19,8 +19,11 @@
     OIL_LEVEL("OilLevel"),
     OIL_PRESSURE("OilPressure"),
     BATTERY_PREDICTION("BatteryPredictiveMaintenance"),
-    AIR_FILTER_PREDICTION("AirFilterPredictiveMaintenance");
-
+    AIR_FILTER_PREDICTION("AirFilterPredictiveMaintenance"),
+    SEATBELT("Seatbelt"),
+    LOW_BREAK_FLUID_WARNING("LowBrakeFluidWarning"),
```
Comment (dacheng81) [inline mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ObjectType.java:24]: @SvitlanaLitovchenko spelling error. Should be "brake".

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/values/HybridBatteryChargeLevel.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package io.moj.java.sdk.model.values;
+
+import com.google.gson.annotations.SerializedName;
+import io.moj.java.sdk.model.enums.RiskSeverity;
+
+public class HybridBatteryChargeLevel extends VehicleSimplePropertyDataModel {
+    @SerializedName(value = "RiskSeverity", alternate = "riskSeverity")
+    private RiskSeverity RiskSeverity;
+
+    public io.moj.java.sdk.model.enums.RiskSeverity getRiskSeverity() {
```
Comment (dacheng81) [inline mojio-sdk-model/src/main/java/io/moj/java/sdk/model/values/HybridBatteryChargeLevel.java:10]: Is the full package name necessary here?

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ObjectType.java (hunk 1 above)
Comment (SvitlanaLitovchenko) [reply mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ObjectType.java]: > @SvitlanaLitovchenko spelling error. Should be "brake".

@dacheng81 fixed

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/values/HybridBatteryChargeLevel.java (hunk 2 above)
Comment (SvitlanaLitovchenko) [reply mojio-sdk-model/src/main/java/io/moj/java/sdk/model/values/HybridBatteryChargeLevel.java]: > Is the full package name necessary here?

@dacheng81 I have re-checked - looks like it is redundant, though IDE doesn't display it. Removed package name. build and tests are successful.

**Review by dacheng81 — APPROVED** [review]

**Pull request merged**